packages/react/src/generators/application/application.spec.ts

```diff
@@ -311,14 +317,6 @@ describe('app', () => {
     );
   });
 
-  it('should setup jest with babel-jest support', async () => {
```

Comment:
Is there a reason this test is deleted?

Reply:
Good catch. No idea how that happened. I'll have a look and replace it.

Reply:
Added back.